config/environments/test.rb

```diff
@@ -1,47 +1,16 @@
 Rails.application.configure do
-  # Settings specified here will take precedence over those in
-  # config/application.rb.
-
-  # The test environment is used exclusively to run your application's
-  # test suite. You never need to work with it otherwise. Remember that
-  # your test database is "scratch space" for the test suite and is wiped
-  # and recreated between test runs. Don't rely on the data there!
   config.cache_classes = true
-
-  # Do not eager load code on boot. This avoids loading your whole application
-  # just for the purpose of running a single test. If you are using a tool that
-  # preloads Rails for running tests, you may have to set it to true.
   config.eager_load = false
-
-  # Configure static file server for tests with Cache-Control for performance.
-  config.serve_static_files = true
-  config.static_cache_control = "public, max-age=3600"
-
-  # Show full error reports and disable caching.
+  config.serve_static_files   = true
+  config.static_cache_control = 'public, max-age=3600'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.